test/e2e/specs/specs-playwright/wp-signup__free.ts

```diff
@@ -0,0 +1,138 @@
+import {
+	DataHelper,
+	DomainSearchComponent,
+	GutenbergEditorPage,
+	LoginPage,
+	NewPostFlow,
+	setupHooks,
+	UserSignupPage,
+	SignupPickPlanPage,
+	AccountSettingsPage,
+	AccountClosedPage,
+	BrowserManager,
+	EmailClient,
+	NavbarComponent,
+	MeSidebarComponent,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe.skip( DataHelper.createSuiteTitle( 'Signup: WordPress.com Free' ), function () {
+	const inboxId = DataHelper.config.get( 'inviteInboxId' ) as string;
+	const username = `e2eflowtestingeditor${ DataHelper.getTimestamp() }`;
+	const email = DataHelper.getTestEmailAddress( {
+		inboxId: inboxId,
+		prefix: username,
+	} );
+	const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+	const blogName = DataHelper.getBlogName();
+
+	let page: Page;
+	let domainSearchComponent: DomainSearchComponent;
+	let gutenbergEditorPage: GutenbergEditorPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Signup', function () {
+		it( 'Navigate to Signup page', async function () {
+			const loginPage = new LoginPage( page );
+			await loginPage.clickSignup();
+		} );
+
+		it( 'Sign up as new user', async function () {
+			const userSignupPage = new UserSignupPage( page );
+			await userSignupPage.signup( email, username, signupPassword );
+		} );
+
+		it( 'Select a free .wordpress.com domain', async function () {
+			domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( blogName );
+			await domainSearchComponent.selectDomain( '.wordpress.com' );
+		} );
+
+		it( 'Select WordPress.com Free plan', async function () {
+			const signupPickPlanPage = new SignupPickPlanPage( page );
+			await signupPickPlanPage.selectPlan( 'Free' );
+		} );
+	} );
+
+	describe( 'Interact', function () {
+		it( 'Start a new post', async function () {
+			const newPostFlow = new NewPostFlow( page );
+			await newPostFlow.newPostFromNavbar();
+		} );
+
+		it( 'Enter post title', async function () {
+			gutenbergEditorPage = new GutenbergEditorPage( page );
+			const title = DataHelper.getRandomPhrase();
+			await gutenbergEditorPage.enterTitle( title );
+		} );
+
+		it( 'Publish post', async function () {
+			await gutenbergEditorPage.publish( { visit: false } );
+		} );
+
+		it( 'Return to Home dashboard', async function () {
+			// Temporary workaround due to https://github.com/Automattic/wp-calypso/issues/51162.
+			// Conditional can be removed once fixed.
+			if ( BrowserHelper.getViewportName() === 'mobile' ) {
```

Comment:
This method name changed - should now be `getTargetDeviceName()`

Reply:
Ahh yes, thanks!